src/components/Approval/Approval.tsx

```diff
@@ -209,6 +239,20 @@ export const Approval = () => {
                   <Text translation='common.reject' />
                 </Button>
               )}
+              <Divider my={4} />
+              <Row>
+                <Row.Label>
+                  <Text color='gray.500' translation='trade.estimatedGasFee' />
+                </Row.Label>
+                <Row.Value textAlign='right'>
+                  <RawText>
+                    {toFiat(
+                      bnOrZero(fee).times(fiatRate).times(selectedCurrencyToUsdRate).toNumber(),
```

Comment:
I know this is just shuffled around from the previous implementation and we should be safe with a JS number here but prefer `toString()` not to set precedents and avoid uint256 shenanigans if someone copies this LoC on a big BigNumber